actions/custom.js

```diff
@@ -118,7 +115,7 @@ const useDateObj = (date) => {
   return dateObj
 }
 
-export const normalizeTimeString = (time, formatString = null) => {
+export const normalizeTimeString = (time, formatString = null, timezone) => {
```

Comment:
Was there a reason why this timezone is passed as parameter?

Reply:
remove